starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/setup/CustomSQLiteOpenHelper.java

```diff
@@ -0,0 +1,76 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2.setup;
+
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by justinwells on 10/25/16.
+ */
+
+public class CustomSQLiteOpenHelper extends android.database.sqlite.SQLiteOpenHelper {
+    private static final String DATABASE_NAME = "SHOPPING_DB";
+    private static final int DATABASE_VERSION = 7;
+    public static final String TABLE_NAME = "SHOPPING_LIST";
+    public static final String COL_ID = "id";
+    public static final String COL_ITEM = "ITEM_NAME";
+    private static CustomSQLiteOpenHelper sInstance;
```

Comment:
Might be good to declare this variable after all the constants after a blank line, just so it's easy to see. Not a big deal though.